assets/javascripts/modules/checkout/checkout.js

```diff
@@ -0,0 +1,151 @@
+define(['$',
+        'bean',
+        'modules/checkout/form-elements',
+        'modules/checkout/validations',
+        'modules/checkout/email-check'
+    ],
+    function ($,
+              bean,
+              form,
+              validations,
+              emailCheck
+    ) {
+
+    var $FIRST_NAME = form.$FIRST_NAME,
+        $LAST_NAME = form.$LAST_NAME,
+        $EMAIL = form.$EMAIL,
+        $ADDRESS1 = form.$ADDRESS1,
+        $ADDRESS2 = form.$ADDRESS2,
+        $ADDRESS3 = form.$ADDRESS3,
+        $POSTCODE = form.$POSTCODE,
+        $ACCOUNT = form.$ACCOUNT,
+        $SORTCODE1 = form.$SORTCODE1,
+        $SORTCODE2 = form.$SORTCODE2,
+        $SORTCODE3 = form.$SORTCODE3,
+        $HOLDER = form.$HOLDER,
+        $MANUAL_ADDRESS = form.$MANUAL_ADDRESS,
+        $FULL_ADDRESS = form.$FULL_ADDRESS,
+        $YOUR_DETAILS_SUBMIT = form.$YOUR_DETAILS_SUBMIT,
+        $PAYMENT_DETAILS_SUBMIT = form.$PAYMENT_DETAILS_SUBMIT,
+        $REVIEW_NAME = form.$REVIEW_NAME,
+        $REVIEW_ADDRESS = form.$REVIEW_ADDRESS,
+        $REVIEW_EMAIL = form.$REVIEW_EMAIL,
+        $REVIEW_ACCOUNT = form.$REVIEW_ACCOUNT,
+        $REVIEW_SORTCODE = form.$REVIEW_SORTCODE,
+        $REVIEW_HOLDER = form.$REVIEW_HOLDER,
+        $SMALLPRINT = form.$SMALLPRINT,
+        $FIELDSET_YOUR_DETAILS = form.$FIELDSET_YOUR_DETAILS,
+        $FIELDSET_PAYMENT_DETAILS = form.$FIELDSET_PAYMENT_DETAILS,
+        $FIELDSET_REVIEW = form.$FIELDSET_REVIEW,
+        $EDIT_YOUR_DETAILS = form.$EDIT_YOUR_DETAILS,
+        $EDIT_PAYMENT_DETAILS = form.$EDIT_PAYMENT_DETAILS,
+
+        FIELDSET_COLLAPSED = 'fieldset--collapsed',
```

Comment:
My Preference is towards adding a state class here, e.g., `is-collapsed` as I find it a nice way of signifying that this has been added by some JS / is a behaviour of the component.